src/web/components/label/compliancestate.jsx

```diff
@@ -0,0 +1,91 @@
+/* SPDX-FileCopyrightText: 2024 Greenbone AG
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+import styled from 'styled-components';
+
+import useTranslation from 'web/hooks/useTranslation';
+import {styledExcludeProps} from 'web/utils/styledConfig';
+import Theme from 'web/utils/theme';
+
+const Label = styledExcludeProps(styled.div, [
```

Comment:
We can avoid using `styledExcludeProps` by using the prefix  `$` before the prop.  
Example https://github.com/greenbone/gsa/commit/7e8e2fa503fc49399707c5838f442d81c54d5efb#diff-0ef3336698388f322d6005baff948cc2780f87205c2d8df9c7c31e7972e2ee3dR46